### DISCUSSION TITLE
How to integrate with Nuitka?

### DISCUSSION THREAD

**Discussion by ono24id** [pyscaffold/pyscaffold#647] [Dev Discussions]
How to integrate with nuitka?
https://github.com/Nuitka/Nuitka

on the README.rts they said we can force "python setup.py bdist_wheel" using nuitka compiler if we put the following line

```
[metadata]
build_with_nuitka = True
```

In my project with Pyscaffold that not work, I wonder if pyscaffold treat setup.cfg with different way?

**Comment by abravalheri** [comment]
Hi @hericah , have you tried adding "nuitka" as a build-system dependency?
You can do that via the `[build-system] requires` configuration in your `pyproject.toml` file.
